etc/config-template.toml

```diff
@@ -708,7 +708,7 @@
 ## lower levels. When this is set, `block-based-bloom-filter` will be ignored.
 ## Only effective for `format-version` >= 5.
 ## Disabled by default.
-# ribbon-filter-above-level = 0
+## ribbon-filter-above-level = 0
```

Comment:
```suggestion
# ribbon-filter-above-level = 0
```

Reply:
It fails test `config::tests::test_config_template_is_valid`, because `= 0` means ribbon filter is enabled.